### DISCUSSION TITLE
Voyager Platform Scam - Stealing crypto

### DISCUSSION THREAD

**Discussion by ercanerberk** [WalletConnect/walletconnect-monorepo#1508] [Developer Support] [answered]
Hello

I have invested in Voyager VGX platform and have been following @investvoyager twitter account.
I dm'd a fake account looking like Voyager Support on the comment sections which instructed me to connect my wallet to REDACTED_FOR_YOUR_SAFETY so that they can later get my VGX crypto assets out to this wallet.

That twitter account had a bot that was checking whether I was still typing or not so it's a very sophisticated scammer. Who knows how to setup a website, bot to get people interested in communicating with them and convincing them to give their funds to them.

Stupid as I am, after several days, I decided to give this a try, which took all my assets to that project on wallet connect. 

I hope WalletConnect creates policies to return such money transactions as current banking/credit card companies do. Meanwhile please stop this sophisticated thief so that this doesn't happen to anyone going forward.

Transaction details:
https://etherscan.io/tx/0xdfb6a8f431fdb530ae775b427d7de3f6cc8de3af42ba17693400cd5bc59837a8

![image](https://user-images.githubusercontent.com/19536289/193899510-9fb67896-a5b9-40de-afec-a3682caa2772.png)
Thank you

**Comment by bkrem** [comment, marked as answer]
Hi @ercanerberk,

Having funds stolen is always a horrible situation, I'm sorry to hear that happened to you.

> I hope WalletConnect creates policies to return such money transactions as current banking/credit card companies do. Meanwhile please stop this sophisticated thief so that this doesn't happen to anyone going forward.

WalletConnect simply connects a wallet and a dapp to each other, and acts as a relay for messages between them, without ever touching or having control over user funds. 
Given this, there isn't anything we can do in this case unfortunately. The path of action lies with the respective token team (VGX) and other stakeholders in the respective blockchain ecosystem to blacklist the attacker's stolen funds or return them.

Best of luck and I hope this can be remedied for you.

**Reply by chadyj** [reply]
Adding to what @bkrem said, have a look at our FAQ for some safety tips
https://explorer.walletconnect.com/faq#safety
